_src/lib/node_cache.coffee

```diff
@@ -39,8 +40,37 @@ module.exports = class NodeCache extends EventEmitter
 			errorOnMissing: false
 			# whether values should be deleted automatically at expiration
 			deleteOnExpire: true
+			# enable legacy callbacks
+			enableLegacyCallbacks: false
 		, @options )
 
+		# generate functions with callbacks (legacy)
+		if (@options.enableLegacyCallbacks)
+			_forEach([
+				"get",
+				"mget",
+				"set",
+				"del",
+				"ttl",
+				"getTtl",
+				"keys",
+				"has"
+			], (methodKey) =>
+				# reference real function
+				oldMethod = @[methodKey]
+				@[methodKey] = (...args, cb) =>
+					# return a callback if cb is defined and a function
+					if (typeof cb is "function")
+						try
+							res = oldMethod(...args)
+							cb(null, res)
+						catch err
+							cb(err)
+					else
+						return oldMethod(...args, cb)
+					return
+			)
```

Comment:
explicit return for `(...args, cb) =>` function is missing here.

Reply:
oh sry I meant this function: `(methodKey) =>`

Reply:
commit #64fbb03 fixes this